priam/src/main/java/com/netflix/priam/aws/S3FileSystem.java

```diff
@@ -128,12 +137,29 @@ private long uploadMultipart(AbstractBackupPath path, Instant target)
         if (logger.isDebugEnabled())
             logger.debug("Uploading to {}/{} with chunk size {}", prefix, remotePath, chunkSize);
         File localFile = localPath.toFile();
-        InitiateMultipartUploadRequest initRequest =
-                new InitiateMultipartUploadRequest(prefix, remotePath)
-                        .withObjectMetadata(getObjectMetadata(localFile));
-        String uploadId = s3Client.initiateMultipartUpload(initRequest).getUploadId();
+
+        CreateMultipartUploadRequest.Builder initRequestBuilder = CreateMultipartUploadRequest.builder()
+                .bucket(prefix)
+                .key(remotePath);
+
+        // Add metadata
```

Comment:
There are two types of comments worth writing:

1. Explaining why something was implemented in a way that isn't the simplest possible.
2. Javadoc for automated documentation.

Anything explaining what the subsequent code does (redundant as it is obvious by inspection), any links (rot-prone), why something is present (because it is totally essential) should just be omitted as they make it harder to parse the code.

Please delete all comments you have added.